Lib/fontTools/ttLib/tables/otConverters.py

```diff
@@ -426,6 +428,27 @@ def fromString(value):
 	def toString(value):
 		return fl2str(value, 14)
 
+class Angle(F2Dot14):
+	factor = 1.0/(1<<14) * 180  # 0.010986328125
+	@staticmethod
+	def wrapAround(angle):
+		# Upon compile/dump, normalize angles that span beyond a full circle.
+		# We want to keep the orientation after modulo: e.g. -415 => -55, not 305
+		return angle % math.copysign(360, angle)
+	def read(self, reader, font, tableDict):
+		return super().read(reader, font, tableDict) * 180
+	def write(self, writer, font, tableDict, value, repeatIndex=None):
+		value = self.wrapAround(value)
```

Comment:
Why do you modify value (by calling `wrapAround`) before writing? I don't understand.

Reply:
The reason I'm objecting is that it can hide bugs / change semantics. If you build masters this way then merge them as variable, this `wrapAround` would silently change the semantics of the rotation. Better fail than corrupt.

Reply:
it's common for fonttools to auto-update things on compile. I don't believe it's changing semantics, because wrapAround keeps the sign while doing modulo operation, so e.g. -415 is compiled as -55, not 305.

Reply:
but I can change it if you prefer and we raise an exception when angle >= 360 or <= -360. We'll have to move the wrapping upstream

Reply:
> I don't believe it's changing semantics, because wrapAround keeps the sign while doing modulo operation, so e.g. -415 is compiled as -55, not 305.

I don't see what significance the -55 vs 305 has.

Here's my deal: imagine upstream sources produce a rotation of 300 in one master, and 390 in another master. Going from first master to second, you expect a 90degree rotation counterclockwise.

But if you silently encode 390 as 30, then the rotation will become 270degrees rotation clockwise. That's the semantic change I'm talking about.

Better err than allow this.

Reply:
I don't even think you should do any wrapping upstream.

Reply:
If you know you are compiling a static font wrap is safe (though ... then your sources are odd and perhaps you should fix those!). Which I guess means I agree, don't wrap here, error :)

Reply:
> I don't see what significance the -55 vs 305 has.

i would think one is clockwise, the other counter-clockwise, given the sign (but perhaps that doesn't matter as the end result is the same?)

> imagine upstream sources produce a rotation of 300 in one master, and 390 in another master.. 

if the "sources" are binary TTFs to be merged into a VF (like varLib currently works for the rest of the variable tables) it's technically impossible to encode 390 in the master COLR binary table, it has to be 390 % 360 to fit the limits of F2Dot14-as-angle.

Reply:
i removed the wrapping of the angle in https://github.com/fonttools/fonttools/pull/2359/commits/60f072414478dba95c743e8f6bbdb5792088c71b